src/components/Form.js

```diff
@@ -1,200 +1,184 @@
-	2. Add the component to the switch case
-	3. Update the Examples to show how to use your component
-	4(Optional). If your component needs extra data besides a name/camelCaseName, pass in the data through the
-		options property as individual properties.
-	5. Any state management such as value of the component will be handled by the form.js's
-		state
-	6. Test Component
-
-	________________________________________________________________
-	EXAMPLES:
-		Input:
-			elements = [
-				{
-				placeholder: "First Name",
-					camelCaseName: "firstName",
-					type: "Input",
-					isRequired: true,
-				},
-				{
-				placeholder: "Last Name",
-					camelCaseName: "lastName",
-					type: "Input",
-					isRequired: true,
-				},
-				{
-				placeholder: "Birthday",
-					camelCaseName: "birthday",
-					type: "DatePicker"
-				},
-				{
-				placeholder: "Coolness Level",
-					camelCaseName: "coolnessLevel",
-					type: "PickerInput",
-					isRequired: true,
-					options: {
-						data: ["Super Cool", "Cool", "Uncool", "Super Uncool"]
-					}
-				},
-				{
-				placeholder: "Major",
-					camelCaseName: "major",
-					type: "FilterPicker",
-					isRequired: true,
-					options: {
-						data: ["Mechanical Engineering", "Electrical Engineering", "Computer Science"],
-						type: "single"
-					}
-				}
-			]
-
-			initialValue = [
-				{
-					camelCaseName: "firstName",
-					value: "Steven"
-				},
-				{
-					camelCaseName: "lastName",
-					value: "Perdomo"
-				},
-				{
-					camelCaseName: "birthday",
-					value: "2001-01-01"
-				},
-				{
-					camelCaseName: "coolnessLevel",
-					value: "Super Uncool"
-				},
-				{
-					camelCaseName: "major",
-					value: "Computer Science"
-				}
-			]
-
-			title = ""
-
-			onSubmit = (value) => {
-				console.log(`this is the value: ${value}`)
-			}
-
-			onCancel = () => {
-				console.log("I've been cancelled")
-			}
-
-			visibile = this.state.formVisibility
-
-			changeVisibility = (visible) => {
-				this.setState({ formVisibility: visible })
-			}
-
-			submitButtonName = "Confirm Coolness Level"
-
-		Output:
-			this.state = {
-				firstName: "Haniel",
-				lastName: "Diaz",
-				birthday: null || "6969/04/20",
-				coolnessLevel:  "Super Cool",
-				major: "Computer Science"
-			}
-			onSubmit(this.state)
-
-*/
-const dimension = Dimensions.get("screen");
+import { ScrollView, SafeAreaView, Modal } from "react-native";
+import { Button, ButtonLayout, DatePicker, NavBar, Input, PickerInput, TimePicker, FilterList } from "./";
+
+/**
+ * Types
+ * @typedef {Object} Elements:
+ * 		@property {String} name
+ * 		@property {String} camelCaseName
+ * 		@property {String} type
+ * 		@property {boolean=} isRequired
+ * 		@property {any=} options
+ * @typedef InitialValue:
+ * 		@property {String} name
+ * 		@property {any} value
+ */
+
+/**
+ * Form Component Info
+ * ________________________________________________________________
+ * Props:
```

Comment:
Indent using spaces to maintain indentation regardless of tab size

Reply:
I'd rather not indent with spaces it creates really ugly code. The text editor can usually just matchup indentations even if you use tab. Just didn't see that they were not aligned